pkg/test/env/istio.go

```diff
@@ -59,6 +63,9 @@ var (
 	// IstioOut is the location of the output directory ($TOP/out)
 	IstioOut = verifyFile(ISTIO_OUT, ISTIO_OUT.ValueOrDefaultFunc(getDefaultIstioOut))
 
+	// IstioHelmRepo is the repository containing the Istio Helm charts.
+	IstioHelmRepo = ISTIO_HELM_REPO.ValueOrDefault("https://raw.githubusercontent.com/istio/istio.io/master/static/charts")
```

Comment:
This points to the packages statically and manually built a week or two ago. These files are not updated regularly and may not have the changes in the actual config files.

We now build these in daily releases. E.g. https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/master-20181026-09-15/charts/

Should we use those instead?

Also, I suppose these files will eventually be branch specific at some point, so it may be a good idea to take that into consideration now instead of hard-coding to one location.

Reply:
Not sure why we're trying to read from an external repository. What is wrong with reading things from local repository?